src/utils/defineTransform.js

```diff
@@ -59,6 +56,10 @@ export default function defineTransform ({
         objectName: path.parentPath.value.left.object.name,
         propertyName: path.parentPath.value.left.property.name
       };
+    } else if (path.parentPath.value.type === 'Property') {
```

Comment:
This still seems like it might throw an exception if path.parentPath.value is undefined.  I would suggest making a local var before the if/else block:  `var parentPathValueType = path.parentPath && path.parentPath.value && path.parentPath.value.type;`  Then you can use it in each of the predicates.